.gitbook/assets/ReadonlyREST-api.yml

```diff
@@ -194,6 +194,55 @@ paths:
               examples:
                 UnsupportedMediaTypeResponseExample:
                   $ref: "#/components/examples/UnsupportedMediaTypeResponseExample"
+
+  /api/ror/settings/all_tenants:
+    get:
+      summary: Get all tenants for the authorized user
+      description: The endpoint retrieves all tenants associated with the authorized user.
+      tags:
+        - ReadonlyREST Enterprise license endpoints
+        - All Tenants
+      parameters:
+        - $ref: '#/components/parameters/XRorCorrelationIdHeader'
+      responses:
+        200:
+          description: Returns a list of tenants associated with the authorized user.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GetAllTenants'
+              examples:
+                UpdateMainSettingsSuccessResponse:
+                  $ref: "#/components/examples/GetAllTenantsResponse"
+        403:
+          description: The request was forbidden by ReadonlyREST's ACL
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ForbiddenRorResponse'
+              examples:
+                GetAllTenantsViaApiEnterpriseOnly:
+                  $ref: "#/components/examples/GetAllTenantsViaApiEnterpriseOnly"
+
+  /api/example:
```

Comment:
I'm not sure about it. TBH, I would be confused about what it demonstrates. 
The header can be passed to any Kibana REST API endpoints. So, maybe it's better to show some KBN endpoint here instead?

Reply:
> So, maybe it's better to show some KBN endpoint here instead?

I had the same feeling about a single Kibana endpoint; it would also be confusing, why we described only this single endpoint, and is it a Kibana endpoint or not. I bet there is a high chance that, user will just ignore something like `api/index_management/indices`, because he will think that it's our ROR API endpoint, not something to demonstrate the header.

So maybe something like `api/*` will pay more attention?

Reply:
If we add some proper disclaimer in the description (explaining what we try to show), I think we can do it this way.

Reply:
> If we add some proper disclaimer in the description (explaining what we try to show), I think we can do it this way.

Do you point to `api/*` here?

WDYT about this description?

```
summary: Example endpoint demonstrating multi-tenancy with the x-ror-tenancy-id header
description: >-
  This endpoint represents how all Kibana API requests can use the x-ror-tenancy-id header 
  for multi-tenancy support. The value should match one of your configured group IDs in the ACL. 
  If not specified, the first group defined in the ACL for the user is used as the default value.
```

Reply:
Done ✔️